src/gazebo_airspeed_plugin.cpp

```diff
@@ -84,7 +84,7 @@ void AirspeedPlugin::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
       boost::bind(&AirspeedPlugin::OnUpdate, this, _1));
 
   airspeed_pub_ = node_handle_->Advertise<sensor_msgs::msgs::Airspeed>("~/" + model_->GetName() + "/airspeed", 10);
-  wind_sub_ = node_handle_->Subscribe("~/" + model_->GetName() + "/wind", &AirspeedPlugin::WindVelocityCallback, this);
+  wind_sub_ = node_handle_->Subscribe("~/world_wind", &AirspeedPlugin::WindVelocityCallback, this);
```

Comment:
Is this factoring in the wind direction correctly?

Reply:
As far as I am concerned yes, it used to work correctly before but stopped subscribing to the wind topic with the latest commit